staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -175,26 +178,49 @@ type batchBackend struct {
 	// Receiving maxBatchSize events will always trigger a send, regardless of
 	// if this amount of time has been reached.
 	maxBatchWait time.Duration
+
+	// Channel to signal that the main routine stopped and no new requests will
+	// be sent to the webhook.
+	shutdownCh chan struct{}
```

Comment:
maybe better: "Channel to signal that the shutdown is completed and no go routine is sending anymore."